ui/pages/confirm-add-suggested-nft/index.scss

```diff
@@ -0,0 +1,124 @@
+.confirm-add-suggested-nft {
+  margin: 16px;
+  box-shadow: 0 4px 16px rgba(0, 0, 0, 0.15);
+
+  &__header {
+    padding-bottom: 8px;
+    border-bottom: 1px solid var(--color-border-muted);
+  }
+
+  &__nft-list {
+    padding: 8px;
+    display: flex;
+    flex-flow: column nowrap;
+    width: 100%;
+  }
+
+  &__nft-list-item {
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__nft-details {
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    justify-content: space-between;
+
+    &--error {
+      opacity: 0.5;
+    }
+
+    margin: 0;
+  }
+
+  &__nft-data {
+    display: flex;
+    align-items: center;
+    padding: 8px;
+  }
+
+  &__nft-description,
+  &__nft-standard {
+    min-width: 0;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  &__nft-image {
+    margin-right: 12px;
+    width: 48px;
+    border-radius: 8px;
+    flex: 0 0 auto;
+  }
+
+  &__nft-image-default {
+    margin-right: 12px;
+    flex: 0 0 auto;
+    width: 48px;
+    height: 48px;
+    border-radius: 8px;
+    padding: 0 !important;
+  }
+
+  &__nft-sub-details {
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: space-evenly;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    width: 100%;
+  }
```

Comment:
A good chunk of these properties can be implemented with the Box component

Reply:
done [here](https://github.com/MetaMask/metamask-extension/pull/19454/commits/19234ca62e07ac3b5a83f7181f4f946e68100b43)